src/renderer/component/common/tooltip.jsx

```diff
@@ -8,6 +8,7 @@ type Props = {
   children: ?React.Node,
   icon: ?boolean,
   direction: string,
+  noPadding?: boolean,
```

Comment:
Is there a name for this property that would describe the _type_ of tooltip this is rather than the styling of it?